android/library/src/main/java/com/liferay/mobile/screens/ddl/model/StringField.java

```diff
@@ -78,21 +82,18 @@ protected StringField(Parcel source, ClassLoader loader) {
 
     @Override
     protected boolean doValidate() {
-        boolean valid = true;
-
         String currentValue = getCurrentValue();
+        currentValue = currentValue.trim();
```

Comment:
isn't current value can be null here? If you are calling this method directly from `initializeValidation`. Maybe I'm missing something like the value is not null always

Reply:
Yep, it was breaking some tests as well. I already fixed it, you can check in this commit:
https://github.com/liferay/liferay-screens/pull/593/commits/cee762fd61d9fc7c04d9efc94f61c26c2c9cc781